modules/shared/partials/working-with-collections.adoc

```diff
@@ -0,0 +1,38 @@
+= Working with Collections
+:nav-title: Collections
+:content-type: howto
+:page-topic-type: howto
+
+[abstract]
+The 3.0 API SDKs all work with all features of Collections and Scopes.
```

Comment:
Probably better to go with 3.x here and in line 10,
although we should probably get some broader opinions on this, too.